examples/select-table-row-by-cell-content/README.md

```diff
@@ -0,0 +1,29 @@
+# Select a Table Row by Cell Content
+
+**Test Code**: [index.js](index.js)
+
+**Tested Page**: [index.html](index.html)
+
+This example shows how to select a table row based on the cell content.
+
+The tested page includes a table with `Id`, `First name` and `Last name` columns and three rows of data.
+A click on the row highlights it with a `highlighted` CSS class.
```

Comment:
A click on the row highlights it with adding to row the `highlighted` CSS class.

Reply:
> A click on the row adds a `highlighted` CSS class to highlight it.

or

> A click on the row adds a CSS class to highlight it.